.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 package/govuk-frontend/
+package/digitalmarketplace/
```

Comment:
I noticed that govuk-frontend does commit the package directory.

What if we didn't add this line and also removed the line above ignoring `package/govuk-frontend/`? Then we wouldn't need https://github.com/alphagov/digitalmarketplace-govuk-frontend/pull/13

Reply:
As discussed offline, we do not want to version compiled code only src code.